common/djangoapps/django_comment_common/tests.py

```diff
@@ -33,7 +33,7 @@ def setUp(self):
             "hacky",
             "hacky@fake.edx.org"
         )
-        self.course_key = SlashSeparatedCourseKey("edX", "Fake101", "2012")
+        self.course_key = CourseLocator("edX", "Fake101", "2012")
```

Comment:
I missed this test file the last time around because it wasn't in a test directory.